core/src/main/java/org/hisp/dhis/android/core/category/CategoryCategoryComboLinkStoreImpl.java

```diff
@@ -46,14 +64,48 @@ public long insert(CategoryCategoryComboLinkModel entity) {
         return executeInsert();
     }
 
+    @Override
+    public int delete() {
+        return databaseAdapter.delete(CategoryCategoryComboLinkModel.TABLE);
+    }
+
+    @Override
+    public boolean delete(@NonNull CategoryCategoryComboLinkModel entity) {
+
+        validate(entity);
+
+        bind(deleteStatement, entity);
+
+        return execute(deleteStatement);
+    }
+
+    @Override
+    public boolean update(@NonNull CategoryCategoryComboLinkModel oldCategoryCategoryComboLinkMode,
+            @NonNull CategoryCategoryComboLinkModel newCategoryCategoryComboLinkMode) {
+
+        validateForUpdate(oldCategoryCategoryComboLinkMode, newCategoryCategoryComboLinkMode);
+
+        bindUpdate(oldCategoryCategoryComboLinkMode, newCategoryCategoryComboLinkMode);
+
+        return execute(updateStatement);
+    }
+
+    @Override
+    public List<CategoryCategoryComboLink> queryAll() {
+        Cursor cursor = databaseAdapter.query(QUERY_ALL_CATEGORY_COMBO_LINKS);
+
+        return mapCategoryCategoryComboLinksFromCursor(cursor);
```

Comment:
this name is large, How only there is one method to map from cursor a name as mapFromCursor is more readable, can you rename it?

Reply:
Sure!